Blog-Website/client/src/components/account/Login.jsx

```diff
@@ -75,7 +75,7 @@ const Login = ({ isUserAuthenticated }) => {
     const [login, setLogin] = useState(loginInitialValues);
     const [signup, setSignup] = useState(signupInitialValues);
     const [error, showError] = useState('');
-    const [account, toggleAccount] = useState('login');
+    const [account, toggleAccount] = useState(true);
```

Comment:
Please rename the state to below for more readability if we are changing string to boolean
const [isSignUp, toggleSignUp] = useState(true);